src/DataSource.ts

```diff
@@ -21,9 +28,22 @@ export class DataSource extends DataSourceApi<CmkQuery> {
   }
 
   async query(dataQueryRequest: DataQueryRequest<CmkQuery>): Promise<DataQueryResponse> {
+    for (const target of dataQueryRequest.targets) {
+      target.requestSpec = replaceVariables(target.requestSpec, dataQueryRequest.scopedVars);
+    }
     return this.getBackend().query(dataQueryRequest);
   }
 
+  async metricFindQuery(query: MetricFindQuery, options?: any): Promise<MetricFindValue[]> {
```

Comment:
`unknown` instead of `any` would be fine here, since we're not using the parameter.